utils/compare/graph.py

```diff
@@ -0,0 +1,306 @@
+import io
+import pprint
+from json import loads
+import pandas as pd
+from search import ElasticService
+import numpy as np
+
+def graph(oMetrics, nMetrics):
+    metrics = []
+    x=[]
+    y=[]
+    for index, row in oMetrics.iterrows():
+        test = "{}-{}".format(row['profile'], row['messageSize'])
+        value = "{}".format(row['throughput'])
+        x.append(value)
+        y.append(test)
+    old = {'y' : x,
+        'x' : y,
+        'name' : 'Previous results average',
+        'type' : 'bar',
+        'orientation' : 'v'}
+    x=[]
+    y=[]
+    for index, row in nMetrics.iterrows():
+        test = "{}-{}".format(row['profile'], row['messageSize'])
+        value = "{}".format(row['throughput'])
+        x.append(value)
+        y.append(test)
+    new = {'y' : x,
```

Comment:
These `x, y` need not necessarily interchange. This implementation can be written in more readable and simple way. More details [here](https://github.com/cloud-bulldozer/cpt-dashboard/blob/1146c0e0f2506e5b5cfd8ea37e379519fa4c0e96/backend/app/api/v1/endpoints/quay/quayGraphs.py#L36-L97).